frontend/src/components/atoms/IconButton/IconButton.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const theme = {
```

Comment:
? Please import this object from theme/MainTheme.js

Reply:
done